firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigContainer.java

```diff
@@ -189,6 +193,38 @@ public boolean equals(Object o) {
     return containerJson.toString().equals(that.toString());
   }
 
+  /** Creates a map where the key is the config key and the value if the experiment description. */
+  private Map<String, JSONObject> createExperimentsMap(JSONArray allExperiments)
```

Comment:
I believe this could be `static`? Though alternatively it could be nice to make it have a similar signature to `createRolloutParameterKeyMap` work as an instance method so we call `this.createExperimentsMap()` and `other.createExperimentsMap()`

Reply:
updated to just use instance vars